### PR TITLE
mrc-1239: Extract hintr strings for translation

### PR DESCRIPTION
**Pull request by richfitz**
This PR pulls out all strings that contain spaces for translation. There might be more strings remaining. Nothing additional has been translated, no features have been added.

Importantly, the only testing that anything sensible is happening is the test of a string in the tests themselves.  If a translation is incorrect (missing an interpolate value, incorrect key) and is not covered by a test, then it will not be obvious at this point.

### CODE REVIEW

**Comment by r-ash** [comment]
One thing we will need to do which is missing from this PR is translate the model options values here
https://github.com/mrc-ide/hintr/blob/mrc-1239/R/model_options.R#L49

**Comment by r-ash** [comment]
> One thing we will need to do which is missing from this PR is translate the model options values here
> https://github.com/mrc-ide/hintr/blob/mrc-1239/R/model_options.R#L49

Actually I take that back, all the options we show here. The ones which are created in this function are coming from the data and so will already be in the right language. Translating some of the yes/no options will come from Naomi.

File: R/filters.R [hunk 1]
```diff
@@ -49,7 +48,7 @@ get_indicator_filters <- function(data, type) {
     "anc" = read_wide_indicator_filters,
     "programme" = read_wide_indicator_filters,
     "survey" = read_long_indicator_filters,
-    stop(sprintf("Can't get indicator filters for data type %s.", type)))
+    stop(t_("filters_cant_get_indicator", list(type = type))))
```
[the remark this answers is not in the document]
Comment (richfitz) [reply R/filters.R]: yeah, it's a good shout. We can definitely do this if its wanted, but agree that another PR would be nice.

Individual words we can also do very straightforwardly in house, and patch up as we see things looking weird

**Comment by richfitz** [comment]
> One thing we will need to do which is missing from this PR is translate the model options values here
/R/model_options.R@mrc-1239#L49 

how does this sit with the naomi template translation of that data?

**Pull request merged**